module/VuFindSearch/src/VuFindSearch/Backend/Solr/Connector.php

```diff
@@ -130,13 +130,9 @@ public function __construct(
         $this->url = $url;
         $this->map = $map;
         $this->uniqueKey = $uniqueKey;
-        if ($cf instanceof HttpClient) {
-            $this->clientFactory = function () use ($cf) {
-                return clone $cf;
-            };
-        } else {
-            $this->clientFactory = $cf;
-        }
+        $this->clientFactory = $cf instanceof HttpClient ? function () use ($cf) {
+            return clone $cf;
+        } : $cf;
```

Comment:
Using an arrow function makes this more readable:

```suggestion
        $this->clientFactory = $cf instanceof HttpClient
            ? fn () => clone $cf
            : $cf;
```